.php_cs.dist

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+require_once './vendor/autoload.php';
+
+use Nextcloud\CodingStandard\Config;
+
+$config = new Config();
+$config
+	->getFinder()
+//	->ignoreVCSIgnored(true)
```

Comment:
That one fails for me locally when using the phar, did you also encounter an issue with that @ChristophWurst

Reply:
No, I have not. But you won't need it in an app. I added it so server so external apps are excluded. Bur it doesn't work anyway :)